github/repos_releases.go

```diff
@@ -188,12 +188,12 @@ func (s *RepositoriesService) ListReleaseAssets(owner, repo string, id int, opt
 		return nil, nil, err
 	}
 
-	assets := new([]*ReleaseAsset)
-	resp, err := s.client.Do(req, assets)
+	var assets []*ReleaseAsset
+	resp, err := s.client.Do(req, &assets)
 	if err != nil {
 		return nil, resp, nil
```

Comment:
I spotted this. It also happens in one more place in this file. See https://github.com/google/go-github/pull/63#pullrequestreview-19332985.

Do you want to include a fix for that in this PR @gmlewis?